src/main/java/com/jwplayer/southpaw/util/ByteArraySet.java

```diff
@@ -585,6 +590,23 @@ public boolean retainAll(Collection<?> c) {
         throw new NotImplementedException();
     }
 
+    @Override
+    public boolean equals(Object object) {
+        if (object == null || !(object instanceof ByteArraySet)) {
```

Comment:
intellij is telling me that `object == null` is not necessary as it's covered by the `instanceof` check

Reply:
Will add a test to make sure, thank you!